src/app/common-ui/Popup/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from './style.scss';
+import PropTypes from 'prop-types';
+
+import CloseButton from '../../../assets/images/close-button.png';
+
+import cx from 'classnames';
+
+function Popup(props) {
+  const { onClose, title, content, buttonText, variant, onButtonClick } = props;
+
+  return (
+    <div className={styles.overlay}>
+      <div className={styles.container}>
+        <div className={styles.titleBar}>
+          <div
+            className={cx(
+              styles.title,
+              variant === 'success' ? styles.titleColorSuccess : styles.titleColorFailed,
```

Comment:
By right, it should be

```
variant === 'success'
? styles.titleColorSuccess
: variant === 'failed'
? styles.titleColorFailed
: styles.titleColorDefault,
```

`variant` should be an optional prop.

Please improve later.